sse2neon.h

```diff
@@ -4296,6 +4296,38 @@ FORCE_INLINE __m128 _mm_cvt_pi2ps(__m128 a, __m64 b)
                      vget_high_f32(vreinterpretq_f32_m128(a))));
 }
 
+// Convert packed single-precision (32-bit) floating-point elements in a to
+// packed 32-bit integers, and store the results in dst.
+//
+//   FOR j := 0 to 1
+//       i := 32*j
+//       dst[i+31:i] := Convert_FP32_To_Int32(a[i+31:i])
+//   ENDFOR
+//
+// https://software.intel.com/sites/landingpage/IntrinsicsGuide/#text=_mm_cvt_ps2pi
+FORCE_INLINE __m64 _mm_cvt_ps2pi(__m128 a)
+{
+#if defined(__aarch64__)
+    return vreinterpret_m64_s32(
+        vget_low_s32(vcvtnq_s32_f32(vreinterpretq_f32_m128(a))));
+#else
+    int32_t res[2];
+    float32_t *data = (float32_t *) &a;
+    for (int i = 0; i < 2; i++) {
```

Comment:
Is it better to unfold it?

Reply:
For performance, it is.
I'll unfold it.